Dispatcher/Networking/UserDefaultsManager.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class UserDefaultsManager: NSObject {
```

Comment:
Please remove this class from the networking dir since it's not network related.
For local db files we use directory called persistence.

Reply:
Fixed
@abenartz